README.rst

```diff
@@ -27,6 +27,48 @@ INGInious can be used as an external grader for EDX. The course `Paradigms of Co
 .. _Docker: https://www.docker.com/
 .. _Paradigms of Computer Programming - Fundamentals: https://www.edx.org/course/louvainx/louvainx-louv1-1x-paradigms-computer-2751
 
+How to install?
+---------------
+
+Simply run:
+
+.. code-block::
+
+   $ REGISTRY=ghcr.io VERSION=latest docker-compose up --build
```

Comment:
On modern docker installations, `compose` is available as a plugin and is run using :

```suggestion
   $ REGISTRY=ghcr.io VERSION=latest docker compose up --build
```

Reply:
Fixed in https://github.com/UCL-INGI/INGInious/pull/882/commits/b34e3e3dbd50df5dada801947ff4ba07e892a7c4